.storybook/webpack.config.js

```diff
@@ -51,6 +52,11 @@ module.exports = (storybookBaseConfig, configType) => {
         },
       ],
     },
+    {
```

Comment:
What are the pros and cons over this and using a babel plugin to remove TS types especially as this is nly for the storybook.

Reply:
I started but didn't really get anywhere. Don't worry about that

Reply:
You mean the babel preset not living up to its promises? I've seen it being used elsewhere that's why I hoped it being able to simplifying things.

Reply:
Sorry I'm bad at responding to the correct thing I meant in response to 

>> Didn‘t @montezume also start this (I remember him mentioning it)? Have you coordinated with him how to „attack“ and go about the setup?